fe/projects/Seller.Portal/src/areas/Orders/components/OrderForm/OrderForm.js

```diff
@@ -493,6 +509,15 @@ function OrderForm(props) {
                                         setMoreInfo(e.target.value);
                                     }} />
                             </div>
+                            <div className="column is-2 is-flex is-align-items-flex-end">
```

Comment:
**suggestion:** Checkbox should be displayed only when selected product is from stock